otx/algorithms/detection/utils/utils.py

```diff
@@ -98,3 +99,26 @@ def generate_label_schema(label_names: Sequence[str], label_domain: Domain = Dom
     label_schema.add_group(exclusive_group)
     label_schema.add_group(empty_group)
     return label_schema
+
+
+def get_det_model_api_configuration(label_schema: LabelSchemaEntity, task_type: TaskType, confidence_threshold: float):
+    """Get ModelAPI config."""
+    omz_config = {}
+    if task_type == TaskType.DETECTION:
+        omz_config[("model_info", "model_type")] = "ssd"
+    if task_type == TaskType.INSTANCE_SEGMENTATION:
+        omz_config[("model_info", "model_type")] = "instance_segmentation"
```

Comment:
The current version of Model API supports "MaskRCNN" and "Yolact"  model types for instance segmentation.